src/apps/search-result/search-result.dev.tsx

```diff
@@ -101,7 +101,83 @@ export default {
+      control: { type: "text" }
+    },
+    facetBrowserModalScreenReaderModalDescriptionText: {
+      name: "facet browser screen reader modal description text",
+      defaultValue: "Modal for facet browser",
+      control: { type: "text" }
+    },
+    facetBrowserModalCloseModalAriaLabelText: {
+      name: "facet browser close modal aria label text",
+      defaultValue: "Close facet browser modal",
+      control: { type: "text" }
+    },
+    facetAccessTypesText: {
+      name: "Access types text",
+      defaultValue: "Access types",
+      control: { type: "text" }
+    },
+    facetChildrenOrAdultsText: {
+      name: "Children or adults text",
+      defaultValue: "Children or adults",
+      control: { type: "text" }
+    },
+    facetCreatorsText: {
+      name: "Creators text",
+      defaultValue: "Creators",
+      control: { type: "text" }
+    },
+    facetFictionNonfictionText: {
+      name: "Fiction or nonfiction text",
+      defaultValue: "Fiction or nonfiction",
+      control: { type: "text" }
+    },
+    facetGenreAndFormText: {
+      name: "Genre and form text",
+      defaultValue: "Genre and form",
+      control: { type: "text" }
+    },
+    facetMainLanguagesText: {
+      name: "Main languages text",
+      defaultValue: "Main languages",
+      control: { type: "text" }
+    },
+    facetMaterialTypesText: {
+      name: "Material types text",
+      defaultValue: "Material types",
+      control: { type: "text" }
+    },
+    facetSubjectsText: {
+      name: "Subjects text",
+      defaultValue: "Subjects",
+      control: { type: "text" }
+    },
+    facetWorkTypesText: {
+      name: "Work types text",
+      defaultValue: "Work types",
+      control: { type: "text" }
+    },
+    showResultsText: {
+      name: "Show results text",
+      defaultValue: "Show results",
+      control: { type: "text" }
+    },
+    filterListText: {
+      name: "Filter list text",
+      defaultValue: "Filter list",
+      control: { type: "text" }
+    },
+    clearAllText: {
+      name: "Clear all text",
+      defaultValue: "Clear all",
+      control: { type: "text" }
+    },
+    addMoreFiltersText: {
+      name: "Add more filters text",
+      defaultValue: "+ MORE FILTERS",
```

Comment:
Please do not capitalize text if not needed.

If it is a visual effect then casing should be managed with CSS.